sapl/relatorios/views.py

```diff
@@ -1880,6 +1885,80 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class RelatorioVotacoesNominaisView(RelatorioMixin, VotacoesMultiFormatOutputMixin, FilterView):
+    filterset_class = RelatorioVotacoesNominaisFilterSet
+    template_name = 'relatorios/RelatorioVotacoesNominais_filter.html'
+    relatorio = relatorio_votacao_nominal
+    paginate_by = 20
+
+    fields_base_report = [
+        'votacao_id', 'votacao', 'parlamentar__nome_parlamentar', 'voto'
+    ]
+
+    fields_report = {
+        'csv': fields_base_report,
+        'xlsx': fields_base_report,
+        'json': fields_base_report,
+    }
```

Comment:
Com as recentes refatorações, fica assim:

```suggestion
    export_fields = [
        'votacao_id', 'votacao', 'parlamentar__nome_parlamentar', 'voto'
    ]
```